homeassistant/components/huawei_lte/__init__.py

```diff
@@ -58,6 +58,7 @@
     ATTR_CONFIG_ENTRY_ID,
     CONF_MANUFACTURER,
     CONF_UNAUTHENTICATED_MODE,
+    CONF_UNVERIFIED_HTTPS,
```

Comment:
Maybe `homeassistant.const.CONF_VERIFY_SSL` should be used instead of this :thinking: